packages/kbn-securitysolution-io-ts-list-types/src/common/search_after/index.test.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { exactCheck } from '@kbn/securitysolution-io-ts-utils';
+import { search_after } from '.';
+
+import { pipe } from 'fp-ts/lib/pipeable';
+import { left } from 'fp-ts/lib/Either';
+
+import { foldLeftRight, getPaths } from '@kbn/securitysolution-io-ts-utils';
+
+describe('pitOrUndefined', () => {
```

Comment:
Same as with pit tests, maybe worth checking for `undefined` being valid.

Reply:
Yeah, adding them now, thanks